exla/c_src/iree_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
```

Comment:
note: This will most likely be extracted to a separate project so that we can build the compiler as a shared library and the runtime as a separate, more lightweight one (this is important for nerves usage).

So, probably, don't worry about adding cmake as a requirement to build exla.